docs/AnalyticObserving.md

```diff
@@ -0,0 +1,44 @@
+# Observing Analytics
+
+A event listener can be registered to allow for the host application to see all of the analytics that are being tracked by the Appcues React Native Module.
+
+## Registering the Event Listener
+
+You can register to receive Appcues analytics events using `addListener` with an `eventType` of `analytics` on the `NativeEventEmitter` class:
+
+```js
+import { NativeEventEmitter, NativeModules } from 'react-native';
+
+...
+
+// Add the listener
+const eventEmitter = new NativeEventEmitter(NativeModules.AppcuesReactNative);
+const eventListener = eventEmitter.addListener('analytics', ({analyticType, value, properties, isInternal}) => {
+    // analyticType: Enum string indicating the type of the analytic. Possible values listed below.
+    // value: String containing the primary value of the analytic being tracked. For events - the event name, for screens - the screen title, for identify - the user ID, for group - the group ID.
+    // properties: Object containing the properties that provide additional context about the analytic.
+    // isInternal: Boolean indicating if the analytic was internally generated by the SDK, as opposed to passed in from the host application.
+    console.log(analyticType, value, properties, isInternal);
+});
+
+...
+
+//Removes the listener
+eventListener.remove();
+ ```
+
+## Using the Analytics Tracking Data
+
+There are four different possible values for `analyticType` and two possible values for `isInternal`:
+
+| `analyticType` | `isInternal` | SDK call              |
+| -------------- | ------------ | --------------------- |
+| `EVENT`        | `true`       | N/A                   |
+| `EVENT`        | `false`      | `Appcues.track()`     |
+| `SCREEN`       | `false`      | `Appcues.screen()`    |
+| `IDENTIFY`     | `false`      | `Appcues.identitfy()` |
+| `GROUP`        | `false`      | `Appcues.group()`     |
+
+Events with `isInternal == true` correspond to internal SDK events - these capture anything that is generated automatically inside of the Appcues SDK, including flow events and session events.
+
+All other analytics - these are all the screens, events, user or group identities that are passed into the SDK from the host application have `isInternal == false`.
```

Comment:
This is a nice summary of the output! I think better than what we have on the iOS/Android SDK side even.  We don't have such a table there do we?

Reply:
Thanks! Yep, the table is new here. I was struggling with copy/pasting because those docs mention "two key types of Appcues analytics tracking data" and then there's `analyticType` plus the RN `eventType` and that's just too many "type" differences to keep straight.